backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/snapshots/RemoveSnapshotCommand.java

```diff
@@ -434,13 +434,13 @@ protected boolean validate() {
      * @return True if there is enough space in all relevant storage domains. False otherwise.
      */
     private boolean validateStorageDomains() {
-        MultipleStorageDomainsValidator storageDomainsValidator = getStorageDomainsValidator(getStoragePoolId(), getStorageDomainsIds());
         if (DisksFilter.filterImageDisks(getSourceImages()).isEmpty()) {
             return true;
         }
 
+        MultipleStorageDomainsValidator storageDomainsValidator = getStorageDomainsValidator(getStoragePoolId(), getStorageDomainsIds());
         return validate(storageDomainsValidator.allDomainsExistAndActive())
-                && validate(storageDomainsValidator.allDomainsWithinThresholds())
+                && (validate(storageDomainsValidator.allDomainsWithinThresholds()) || getParameters().getParentCommand() == ActionType.LiveMigrateDisk)
```

Comment:
Here I had doubts whether to put `getParameters().getParentCommand() == ActionType.LiveMigrateDisk)` first.
Due to lazy evaluation that would skip `validate(storageDomainsValidator.allDomainsWithinThresholds())` for LSM flows, which is obviously more efficient, but on the other side in the current implementation if we fail later validation messages will contain the `ACTION_TYPE_FAILED_DISK_SPACE_LOW_ON_STORAGE_DOMAIN`.
Is it worth it?

Reply:
why would we want to have ACTION_TYPE_FAILED_DISK_SPACE_LOW_ON_STORAGE_DOMAIN if something fails later on?
I also wonder if there's a case we'll end up consuming more space after the execution of RemoveSnapshot - if not, then maybe dropping the allDomainsWithinThresholds check would actually be a better thing to do..

Reply:
So, I'm totally removing the `allDomainsWithinThresholds()` check :)

Reply:
@barpavel I think something went wrong in your rebase - I still see the same state

Reply:
@ahadas no, nothing wrong, I didn't commit it yet.
I tried to test the new code (just to verify it) with the latest master and got stuck.
`engine-setup` doesn't work anymore - it does nothing.
Tried to reinstall from scratch including full delete of deploy and `.m2` folders just to eliminate the chance for some junk/leftovers/irrelevant stuff - didn't help.
And then checked the latest commit that looks very much suspicious, looks like something that waited from April 27 (?!).
```
Author: Yedidyah Bar David <didi@redhat.com>
Date:   Wed Apr 27 17:05:47 2022 +0300

    packaging: setup: Filter from logs secrets from otopi answer files
    
    When running engine-setup with an answer file generated by a previous
    interactive engine-setup, without this patch, we log some secrets
    unfiltered. Fix this:
...
```
I will update the PR, but I didn't test it, though I don't expect any issues with this change.

Reply:
see `[ovirt-devel] [ACTION REQUIRED] engine-setup: getCredentials updated`

Reply:
Thanks Benny, yes, already talked with Yedidyah and saw the email and all solved :)

Reply:
Done.
Removed the `allDomainsWithinThresholds()` check unrelated to parent's command.